tests/simulator/__init__.py

```diff
@@ -55,10 +54,10 @@ def defaultPopulationSettings():
         },
         2: {
             "density": lambda x, y: 1.0 / np.cosh((x - xmax * 0.5)) ** 2
-            + background_particles,
-            "vbulkx": lambda x, y: np.sin(1 * pi_over_xmax * x),
-            "vbulky": lambda x, y: np.sin(1 * pi_over_xmax * x),
-            "vbulkz": lambda x, y: np.sin(1 * pi_over_xmax * x),
+            + background_particles  + (y * .01),
```

Comment:
we would better have periodic in both directions

Reply:
suggestions welcome

Reply:
I gave you a notebook at some point with 2D periodic functions
let me see if I can find it if you don't have it anymore

Reply:
![image](https://user-images.githubusercontent.com/3200931/86588463-06491180-bf8c-11ea-90f0-cd7c58784754.png)

Reply:
I tried to implement this, not sure what I did is right tbh

Reply:
ok, so you've implemented this function, but you're using it for the density... that can't be, because the density needs to always be positive (cannot have negative number of particles per surface unit).
Density profile you could use 
```python
nx = 100
ny = 100
xmax = 11
ymax= 12
dx = xmax/nx
dy = ymax / ny
orix = 0.
oriy = 0.
x = np.arange(nx)*dx + orix
y = np.arange(ny)*dy + oriy
xx, yy = np.meshgrid(x,y,indexing="ij")

nb = 0.3
zz = np.exp(-(xx-0.5*xmax)**2)*np.exp(-(yy-ymax/2.)**2) + 0.3

fig,ax = plt.subplots(figsize=(5,5))
im=ax.pcolormesh(xx,yy,zz)
fig.colorbar(im)
```
![Unknown](https://user-images.githubusercontent.com/3200931/87218799-54785f00-c356-11ea-8773-922d6fc17882.png)

Reply:
reminder?